src/Microsoft.SourceIndexer.Tasks/DownloadStage1Index.cs

```diff
@@ -118,14 +120,15 @@ private void ExecuteCore()
             try 
             {
                 using Stream fileStream = blobClient.OpenRead();
+                using var input = new GZipInputStream(fileStream);
+                using var archive = TarArchive.CreateInputTarArchive(input, Encoding.UTF8);
+                archive.ExtractContents(OutputDirectory, true); // would like this to be false, but SharpZipLib has a bug in 1.3.3
```

Comment:
Consider moving this later to use System.IO.Compression and System.Formats.Tar instead.